core-aws/outputs.tf

```diff
@@ -42,3 +42,12 @@ output "default_db_subnet_group_id" {
 output "default_db_subnet_group_arn" {
   value = var.create_db_subnet_groups ? aws_db_subnet_group.default[0].arn : null
 }
+
+output "iam_assume_groups" {
+  description = "to be fed into the iam_assume_groups input variable of the core-aws-main module"
+  value = {
+    "${var.environment}-admins"     = { roles = [module.iam_assumable_roles.admin_iam_role_arn] },
```

Comment:
Maybe it would be better just to use the keys without the env name here, would make it easier to reference later on other modules. It's the values that matter anyways.

Reply:
the reson why the keys are named like this because this way it's a matter of matter copy and paste directly into to the core-aws-main iam_assume_groups variable, which is a manual process at the moment because it did not seem right to set a terragrunt level dependency on all environments :(